linux_os/guide/services/xwindows/disabling_xwindows/package_xorg-x11-server-common_removed/rule.yml

```diff
@@ -47,7 +47,18 @@ ocil: |-
     The output should be:
     <pre>package xorg-x11-server-common is not installed</pre>
 
+warnings:
+    - general: |-
+        If you use a system with Graphical User Interface (GUI), by removing the package xorg-x11-server-common, it can
+        potentially remove the graphical target and it might bring your system to an inconsistent state where
+        additional configuration can be required to access the system again. Only remove this package if you are sure.
```

Comment:
Warning should be of type `functionality`.

Text should be updated:
````
        The installation and use of a Graphical User Interface (GUI) increases your attack vector and decreases your
        overall security posture. Removing the package xorg-x11-server-common package can
￼        potentially remove the graphical target which might bring your system to an inconsistent state requiring
￼        additional configuration to access the system again. If a GUI is an operational requirement, a tailored profile
        that removes this rule should used before continuing installation.
````

Reply:
Updated accordingly.